### DISCUSSION TITLE
SDR++ on a Raspberry Pi 7" display

### DISCUSSION THREAD

**Discussion by colorflakes** [AlexandreRouma/SDRPlusPlus#1271] [Q&A]
I am trying to get SDR++ setup on a Rasp Pi 4B running the 7" 800x480 touchscreen.  I can't use it because the SDR++ opens and fills beyond the screen so I cannot do anything in the software. The only way to close it is ALT-F4.

Is there a fix to make this work on the small screen?

I've tried a previous version of SDR++ and it fit the screen fine but I had audio issues.

**Comment by dsittner** [comment]
Just fyi, that display blasts broadband RF interference.  I recommend not using it if you are line of sight to any of your antennas.
